### PR TITLE
Backup Node Credentials

### PR DESCRIPTION
**Pull request by PeterChung241**
## Description
Script backs up node credentials for Docker and K8s.

## Affected Dependencies
N/A

## How has this been tested?
Locally. Script to backup credentials tested on both k8s and docker.

## Checklist
- [X] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [X] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [X] My changes are covered by tests

### CODE REVIEW

File: packages/grid/scripts/backup_node_credentials.sh [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    CONTAINER_NAMES=$(docker ps --filter "name=$CONTAINER_NAME_PATTERN" --format "{{.Names}}")
+
+    if [ -z "$CONTAINER_NAMES" ]; then
+        echo "No containers found with the name pattern: $CONTAINER_NAME_PATTERN"
+        exit 1
+    fi
+
+    for CONTAINER_NAME in $CONTAINER_NAMES; do
+        # Define the source path of the credentials.json file in the container
+        SOURCE_PATH="/storage/credentials.json"
```
Comment (yashgorana) [inline packages/grid/scripts/backup_node_credentials.sh]: Hi @PeterChung241 This merged change https://github.com/OpenMined/PySyft/pull/8179 tweaked the path where syft stores data & credentials.
* `/root/data/creds/credentials.json`
* `/root/data/db` is where sqlite db is stored

With this you now have the option of backing up the whole `/root/db`.

Also, once we move from root user `/root/` to a rootless user `/root/syftuser` these paths will change again. So I'll recommend we move `SOURCE_PATH` `FILE_PATH_IN_POD`, etc to a common variable at the start of that scripts. that way it'll be easier to track and change backup paths in the future.

**Comment by shubham3121** [comment]
https://openmined.slack.com/files/U02SLC20VCJ/F062V4P6G7Q/backup_credentials_script.mkv

File: packages/grid/scripts/backup_node_credentials.sh (hunk 1 above)
Comment (PeterChung241) [reply packages/grid/scripts/backup_node_credentials.sh]: Thanks Yash, changes and suggestion have been incorporated.

**Review by shubham3121 — APPROVED** [review]
Great work @PeterChung241 !! This is super cool 🔥 💯

**Pull request merged**